src/molecules/modal/Modal.scss

```diff
@@ -84,20 +87,23 @@ $modal__close-button-icon-transition : $transition-base !default;
     @include mq($screen-xl) {
       width: $modal__content-width\@extra-large;
     }
-
-    &--blank {
-      padding: $reset;
-    }
   }
 
   &__header {
     display: flex;
+    flex-wrap: wrap;
     justify-content: space-between;
     margin: $modal__header-margin;
   }
 
+  &__header &__divider {
```

Comment:
Can it be just `&__divider`?.
In line 125 please change `&-icon` to `&__close-button-icon`

Reply:
Styles presented in `&__header &__divider` are for specific - default - header presentation case. I'll move them directly to divider as a BEM modifier. Let me know if you have any other idea for it.

Also, icon styles have been modified to bit simpler `&__close-icon`.